Project.toml

```diff
@@ -23,7 +23,6 @@ ColorVectorSpace = "0.7, 0.8"
 FileIO = "1"
 ImageAxes = "0.6"
 ImageCore = "0.8.1"
-Libz = "1"
```

Comment:
Compat for `CodecZlib` is needed.

Reply:
Thanks for the review! Are you referring to https://github.com/JuliaLang/Compat.jl? I haven't used this before but will take a look

Reply:
Oh sorry for the inaccuracy.

I mean we need `CodecZlib = "0.6, 0.7"` so that we can tag a new release; registring a new release requires all non-standard libraries having a compat record.

Reply:
yes, 0.6 works!